javascript-practice/src/js/constants/config.js

```diff
@@ -0,0 +1,7 @@
+const API_BASE_URL = 'http://localhost:3000';
+
+const API_ENDPOINTS = {
+  GET_PRODUCTS_URL: `${API_BASE_URL}/products`,
```

Comment:
Only define  `GET_PRODUCTS_URL: `products``

Because you'll add API_BASE_URL in api core, so don't need to add duplicate here